connectors/cli.py

```diff
@@ -91,17 +92,22 @@ def _parser():
 def run(args):
     """Runner"""
     # just display the list of connectors
+    config = EnvYAML(args.config_file)
     if args.action == "list":
         logger.info("Registered connectors:")
-        config = EnvYAML(args.config_file)
         for source in get_data_sources(config):
             logger.info(f"- {source.__doc__.strip()}")
         logger.info("Bye")
         return 0
 
+    loop = get_event_loop(args.uvloop)
+    try:
+        loop.run_until_complete(pre_flight_check(config))
```

Comment:
Why not `await pre_flight_check(config)`?

Reply:
`await` only works in `async` function I think